src/Resources/TortoiseProcENG.rc

```diff
@@ -1697,7 +1697,7 @@ STYLE DS_SETFONT | DS_FIXEDSYS | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "General::Set Extend Menu Item"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    GROUPBOX        "Move to extended context menu (press shift key to show it)",-1,7,7,286,210
+    GROUPBOX        "I don't want to show these itmes... (press shift key to show them)",-1,7,7,286,210

```

Comment:
Maybe better: "Don't show the selected items by default" or "Show the selected items only in the extended xontect menu"; "Show the selected items olny when pressing SHIFT" - Unsure...
Don't know how much space we have...